api/create_release.sh

```diff
@@ -66,6 +76,6 @@ if [ -z "$RM_API_HOST" ]; then
   RM_API_HOST="https://api.bitrise.io"
 fi
 
-release=$(create_release)
-request_error "$release"
-process_create_release_response "$release"
+release_full_resp=$(create_release)
+request_error "$release_full_resp" '/releases'
```

Comment:
I think we should be consistent, so either we use the '/' in the name of the endpoints or not.
Please, remove it from here OR add the missing '/' to the other places where you call the `request_error` method.